src/core/memory.cc

```diff
@@ -1494,18 +1500,24 @@ void* allocate(size_t size) {
     if (size <= sizeof(free_object)) {
         size = sizeof(free_object);
     }
+
+#ifdef SEASTAR_HEAPPROF
+    bool should_sample = get_cpu_mem().should_sample(size);
+#else
+    bool should_sample = get_cpu_mem().should_sample(size);
+#endif
```

Comment:
Good eye. I accept your offer to ignore this nitpick however as rewriting it to fix this will take a few minutes I could spend doing something like improving sesatar compile times.

Reply:
> improving sesatar compile times

Modules will save us. R-right?

Reply:
I will play with modules from the retirement home I'll be in I guess.